.github/workflows/on-remove.yml

```diff
@@ -213,15 +213,21 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-default-region: ${{ steps.setup.outputs.aws-default-region }}
-      - name: Delete the index
-        if: ${{ steps.setup.outputs.crn-on-branch-env == 'true'}}
+      - name: Delete the entity index
+        if: ${{ steps.setup.outputs.crn-contentful-on-branch-env == 'true'}}
```

Comment:
Not sure if it's an overkill, but maybe we could have something similar to what we have in the Contentful teardown script (https://github.com/yldio/asap-hub/blob/ASAP-681-pr-algolia-dedicated-env-label/packages/contentful/scripts/teardown.sh#L1-L11) where we guarantee we're not deleting prod or dev. So here maybe we could check if `steps.setup.outputs.crn-algolia-index` is not `asap-hub_dev` or `asap-hub_prod`